app/helpers/hosts_helper.rb

```diff
@@ -431,4 +431,17 @@ def prioritized_members(list, value_key)
       sort_by { |member| member[:priority] }.
       map { |member_hash| member_hash[value_key] }
   end
+
+  def host_errors(host)
```

Comment:
sounds like this will return only lookup_values errors (and not just errors, but full lookup values data!) so I'd expect a different name, and probably even different component then `ResourceErrors` for it?

Reply:
The main purpose of this method is to collect errors that will be passed to the react component. It currently returns only information about lookup values errors because I know I will need them, but more can be added in the future if there is a need. 

What name would you suggest?